src/client/pages/Login/ChangePassword/ChangePassword.scss

```diff
@@ -0,0 +1,31 @@
+@import 'src/client/style/partials';
+
+.change-password {
+  // Hide labels
+  .form-cell-label {
+    display: none;
+  }
+
+  .form-grid {
+    grid-template-columns: 1fr !important;
+    grid-row-gap: 40px;
+    box-shadow: none;
+  }
+
+  .data-cell {
+    border: none !important;
+  }
+
+  input[type='text'],
+  input[type='password'] {
+    border: $ui-thin-border;
```

Comment:
can we pass bordered to TextInput ?

Reply:
https://github.com/openforis/fra-platform/pull/5453/commits/4fe60fbbb3345a708b507571d54220d5b1f9a744